### ISSUE TITLE
Grapher: Change the default order of countries in Grapher from alphabetical to highest value

### ISSUE THREAD

**Issue by mrwbkrm** [owid/owid-grapher#4202]
## Core problem

Bobbie noticed a quite a few users in his session replays automatically clicking on the first countries in the list when looking at a chart and not really coming up with anything interesting. 

## Proposed solution

After discussing this in the data viz group with Sophia, Marcel and Daniel, we decided that we're in favor of changing the default order of entities to highest value only for charts that have one indicator. This is because in some cases, picking the first or a random indicator from a list will not make much sense to the user. Example [here](https://ourworldindata.org/grapher/number-electoral-democracies-age)

## Context

You can read/watch more about this in [this Slack thread](https://owid.slack.com/archives/C07UTA5N23B/p1730916606433799)

**Comment by danyx23** [comment]
There might be some weird edge cases but we should put up a staging server that does this and see what happens. Marwa and Joe talked about this and had some concerns about some edge cases, get in touch with them before shipping this in any case.

**Comment by mrwbkrm** [comment]
That's right. You can see our concerns outlined in [this Slack thread](https://owid.slack.com/archives/C04VBQ00HQR/p1733930389580069).
